JatApp/AutentificationViewModel.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  AutentificationViewModel.swift
+//  JatApp
+//
+//  Created by Iryna Kopchynska on 24.08.2020.
+//  Copyright © 2020 Iryna Kopchynska. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class AutentificationViewModel {
+    
+    var logIn = LoginAccount()
+    var signIn = CreateAccount()
+
+    func loginToAccount(_ email: String, _ password: String) {
```

Comment:
please add completion handler that will return success with data, that prepared for UI, or failure with error String